modules/mobile/config/routes.rb

```diff
@@ -38,6 +38,7 @@
     get '/dependents/request-decisions', to: 'dependents_request_decisions#index'
     get '/disability-rating', to: 'disability_rating#index'
     get '/enrollment-status', to: 'enrollment_status#show'
+    get '/efolder', to: 'efolder#index'
```

Comment:
I think I could change this to efolder/documents or something along those lines or even /uploaded-documents just to make it more clear what this endpoint actually does and in case there's ultimately a a future endpoint that is more about the meta data of the efolder itself.